lib/endpoint/find-parameters.js

```diff
@@ -191,6 +189,9 @@ function findInBlocks (state) {
     state.blocks.splice(parametersBlockIndex - 1, 2)
 
     state.results.forEach(result => {
+      for (let i = 0; i < params.length; i++) {
+        params[i].description = params[i].description.replace(/]\(\/v3/g, '](https://developer.github.com/v3').replace(/]\(\/apps/g, '](https://developer.github.com/apps').replace(/]\(\/webhooks/g, '](https://developer.github.com/webhooks')
+      }
```

Comment:
Right now you are setting the description in two places, you are missing a few. I would avoid duplication altogether and maybe make a new function like "makeRelativeLinksAbsolute" and run it after all parameters have been found, below the current `addPaginationParameters(state)` call.

What do you think?

Reply:
I hope I solved the problem now. @gr2m please check my PR and verify.